components/gitpod-cli/cmd/env.go

```diff
@@ -213,6 +227,11 @@ func setEnvs(ctx context.Context, scopeUser bool, args []string) error {
 		g.Go(func() error {
 			err = result.client.SetEnvVar(ctx, v)
 			if err != nil {
+				if ferr, ok := err.(*jsonrpc2.Error); ok && ferr.Code == http.StatusForbidden && setEnvScope == envScopeUser {
+					return fmt.Errorf(""+
+						"Can't automatically create Env Var `%s` for security reasons.\n"+
+						"Please create the var manullay under %s/user/variables using Name=%s, Scope=*/*, Value=foobar", v.Name, result.gitpodHost, v.Name)
```

Comment:
```suggestion
						"Please create the var manullay under %s/user/variables using Name=%s, Scope=*/*, Value=foobar", result.gitpodHost, v.Name, v.Name)
```

Reply:
It's correct as-is:
 1. var.Name
 2. host
 3. var.Name
 :+1: